docs/testplan/BGP_IPv6_test_2TierNetwork.md

```diff
@@ -0,0 +1,32 @@
+# Test Objective
```

Comment:
BGP + route scale

Reply:
actually, better split the route scale into another test. for bgp session, we can use 4 x Number of ports as route scale.

Reply:
for route scale test, we can do 40 x number of ports. but in separate test. in the route scale test, we need to check the latency very carefully.